src/extension.ts

```diff
@@ -105,9 +106,19 @@ export function activate(context: vscode.ExtensionContext) {
 
         const showVulnerabilityFoundPrompt = async (msg: string) => {
           const selection = await vscode.window.showWarningMessage(`${msg}. Powered by [Snyk](${registrationURL})`, StatusMessages.FULL_STACK_PROMPT_TEXT);
+          // telemetry
+          let event = {
+            type: 'track', // type of telemetry event such as : identify, track, page, etc.

```

Comment:
is the `type` defined in telemetry package? I feel it is perfect candidate for enum.

Reply:
ya, it is a perfect candidate for enum. but I may not be required in our implementation.